src/fidesops/main.py

```diff
@@ -38,6 +38,14 @@
 def start_webserver() -> None:
     """Run any pending DB migrations and start the webserver."""
     logger.info("****************fidesops****************")
+
+    if logger.getEffectiveLevel() == logging.DEBUG:
+        logger.warning(
+            "WARNING: log level is DEBUG, so sensitive or personal data may be logged. "
+            "Set FIDESOPS__SECURITY__LOG_LEVEL to INFO or higher in production."
+        )
```

Comment:
I'm increasingly of the opinion we should remove the `LOG_PII` variable, and just use this single `LOG_LEVEL` instead. Having DEBUG vs INFO logs is a standard config option most apps support, so it'll be more intuitive to use this.

Reply:
NOTE: I really wanted to DEBUG log all the `ENV` vars at startup too, but by the time this method runs the `FidesopsConfig` has already been loaded (and potentially crashed), so it's too late. Since we're using the config to set the log level simultaneously, it's awkward to use DEBUG logs to see the `ENV` vars... kind of a chicken & egg situation.